docs/community/faq/faq-contributors.md

```diff
@@ -1,5 +1,7 @@
 # First-Time Contributors' Frequently Asked Questions
 
+**TODO**
```

Comment:
What do you think is needed here? An intro to the page, perhaps?

Reply:
Reading through the answers, perhaps we should add "You need to be familiar with..." and then list what prerequisites there are for the package. For instance, a good understanding of machine learning concepts, and NumPy as a package?

Reply:
I just put a todo to include it but show that it's not ready, and I'm not sure we can get this done this week.

About the prerequisites list: Yes, I think that's a good idea!